### ISSUE TITLE
Create a Drupal theme that implements the Italia Design Guidelines

### ISSUE THREAD

**Issue by ruphy** [italia/design-drupal-theme#1]
Create a theme for Drupal based on the [User Interface design guidelines](
https://design-italia.readthedocs.io/it/stable/doc/user-interface.html).

Do include [the design web toolkit](https://github.com/italia/design-web-toolkit) as a dependency, and make sure you write your code in a way which is generic enough so that, later on, it be replaced by another toolkit (e.g. bootstrap) without too much effort.

Some example websites with the correct style
https://developers.italia.it
https://designers.italia.it

**GSoC mentor**: Somebody from in the #design team. If you are unsure who to ask you can check out https://slack.developers.italia.it

**Comment by maurineakwa** [comment]
@ruphy I will like to take on this task. I have read through the project idea and I feel fairly comfortable with taking on this.

**Comment by vaibhavdaren** [comment]
@ruphy  I came across the project titled “Create a Drupal theme that implements the Italia Design Guidelines”, and was delighted to find that my skills meet all of the necessary requirements. I am seeking a challenging but rewarding internship project to work upon this summer which is why I was drawn to this exciting opportunity. I dont know any  Italian will it be ok  for me to work on this project while using google translate to talk with other devs? I translated some of the document guides and was able to comprehend easily.
